data/meson.build

```diff
@@ -0,0 +1,25 @@
+subdir('appstream')
+subdir('completions')
+subdir('control-center')
+subdir('dbus')
+subdir('desktop')
+subdir('gsettings')
+
+if get_option('shell-extension')
```

Comment:
The option name should probably be `gnome-shell` as it doesn't handle only the extension

Reply:
Ah, makes sense, yes.